Documentation/CONFIG.md

```diff
@@ -1,6 +1,6 @@
 # MetroAE Config
 
-MetroAE Config is a template-driven VSD configuration tool. It utilizes the VSD API along with a set of common Feature Templates to create configuration in the VSD. The user will create a yaml or json data file with the necessary configuration parameters for the particular feature and execute simple CLI commands to push the configuration into VSD.
+MetroAE Config is a template-driven VSD configuration tool. It utilizes the VSD API along with a set of common Feature Templates to create configuration in the VSD. The user will create a yaml or excel spreadsheets (.xlsx files) with the necessary configuration parameters for the particular feature and execute simple CLI commands to push the configuration into VSD.
```

Comment:
Like in your PR in metroae-config, please capitalize Excel here and use singular for spreadsheets and files (Excel spreadsheet (.xlsx file))